ol3-viewer/src/ome/ol3/source/Image.js

```diff
@@ -273,7 +283,16 @@ ome.ol3.source.Image = function(options) {
             }
             url += "&maps=" + JSON.stringify(maps);
             url += '&m=' + this.image_model_;
-            url += '&p=' + (this.split_ ? 'split' : this.image_projection_);
+            url += '&p=' +
+                        (this.split_ ?
+                            ome.ol3.PROJECTION['SPLIT'] :
+                            this.image_projection_);
+            if (this.image_projection_ === ome.ol3.PROJECTION['INTMAX'] &&
```

Comment:
how will you identify the projection along T when we introduce it?
it might be better to have INTMAX_Z for example
and nothing could stop us to have both at the same time

Support for T not to be added to this PR something to keep in mind also
we will need to support other projection algorithms

Basically this will need to scale